### PR TITLE
all commands included in a single script file.

### PR DESCRIPTION
**Pull request by yashensha**
- script file created.
- readme.md file modified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,77 @@
-apt install git curl tmux zsh vim-nox bat ncdu ranger neovim python3 python3-pip htop neofetch powerline ifstat nethogs exa
 
-pip3 install --user git+https://github.com/powerline/powerline
+> Run [script.sh](https://github.com/yashensha/dotfiles/blob/main/script.sh) or Execute them seperately.
```
Comment (noormohammedb) [inline README.md]: edit the blob url to ./script.sh

**Comment by noormohammedb** [comment]
overall its good work

File: README.md (hunk 1 above)
Comment (yashensha) [reply README.md]: Merge them.

Comment (noormohammedb) [reply README.md]: i asked you to change the absolute url of the file to relate url. then it can be pointed to the same repo

Comment (yashensha) [reply README.md]: how can i relate link from same repo unless that repo don't have that file?
i don't know how

Comment (yashensha) [reply README.md]: so i removed that link

Comment (yashensha) [reply README.md]: yep i understand now

Comment (yashensha) [reply README.md]: changes made to readme file

**Pull request merged**